packages/cli/src/deprecation/deprecation.service.ts

```diff
@@ -39,25 +42,30 @@ export class DeprecationService {
 			envVar: 'N8N_SKIP_WEBHOOK_DEREGISTRATION_SHUTDOWN',
 			message: `n8n no longer deregisters webhooks at startup and shutdown. ${SAFE_TO_REMOVE}`,
 		},
+		{
+			envVar: 'N8N_RUNNERS_ENABLED',
+			message:
+				'Running n8n without task runners is deprecated. Please set `N8N_RUNNERS_ENABLED=true` to enable task runners. Learn more: https://docs.n8n.io/hosting/configuration/task-runners/',
```

Comment:
Maybe we could mention it's going to become the default:
```suggestion
				'Running n8n without task runners is deprecated. Task runners will be turned on by default in a future version. Please set `N8N_RUNNERS_ENABLED=true` to enable task runners now and avoid potential issues in the future. Learn more: https://docs.n8n.io/hosting/configuration/task-runners/',
```